### DISCUSSION TITLE
chunk: 000000001DD5CA41 replication status: Disconnected

### DISCUSSION THREAD

**Discussion by zhs077** [moosefs/moosefs#376] [Q&A]
moosefs version 3.0.101

mfsmaster log print error : mfsmaster[19430]: (IP1:9422 -> IP2:9422) chunk: 000000001DD5CA41 replication status: Disconnected

**Comment by xandrus** [comment]
Hi,

Basically, it's not an error. It's just information.
Disconnections can happen a specially on a very busy chunk server.

I also recommend upgrading a cluster to the latest stable version 3.113.
Please check our NEW file:
https://github.com/moosefs/moosefs/blob/master/NEWS
